apps/evans/justfile

```diff
@@ -21,6 +24,27 @@ test:
 
 # helpers
 
+[private]
+lang-en:
+  @cp ./components/core/Logo{Friends,}.tsx
+  @cp ./components/core/Logo{Amigos,Alt}.tsx
+
+[private]
+lang-es:
+  @cp ./components/core/Logo{Amigos,}.tsx
+  @cp ./components/core/Logo{Friends,Alt}.tsx
+
+[private]
+web-covers:
+  @mv ../../libs-ts/cover-component/src/Logo{Spanish,Icon,}.tsx .
+  @cp ../../libs-ts/cover-component/src/Logo{.web,}.tsx
+  @cp ../../libs-ts/cover-component/src/LogoSpanish{.web,}.tsx
+  @cp ../../libs-ts/cover-component/src/LogoIcon{.web,}.tsx
+
+[private]
+unweb-covers:
+  @mv ./Logo{,Icon,Spanish}.tsx ../../libs-ts/cover-component/src/
```

Comment:
these 4 private commands are where we handle the logo-related optimizations. these are a bit hacky, but they really make a difference in our speed/lighthouse scores. we were doing the same back in gatsby days, this just really restores that work.